static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -1,157 +1,153 @@
+            inset 0px 0px 0px 3px colour(comment-main-2);
     }
 }
 
-.quiz--results.quiz--knowledge {
+.quiz__answer__icon {
+    position: absolute;
+    left: 1em;
+    top: .9em;
 
-    .quiz__answer--correct .quiz__answer__label {
-        background-color: $correct-answer--background;
+    svg {
+        height: 1em;
+        width: 1em;
     }
+}
 
-    .quiz__answer--selected.quiz__answer--correct .quiz__answer__label {
-        background-color: $user-response-was-right;
-        font-weight: bold;
-        color: colour(neutral-7);
+.quiz__answer--correct {
+    .quiz__answer-text {
+        color: #ffffff;
+        background-color: fade-out(colour(quiz-correct), .4);
     }
 
-    .quiz__answer--selected.quiz__answer--incorrect .quiz__answer__label {
-        background-color: $user-response-was-wrong;
-        font-weight: bold;
-        color: colour(neutral-7);
+    &.quiz__answer--selected {
+        color: #ffffff;
+        background-color: colour(quiz-correct);
     }
 }
 
-.quiz--results.quiz--personality {
-
-    .quiz__answer--selected .quiz__answer__label {
-        background: $correct-answer--background;
+.quiz__answer--incorrect.quiz__answer--selected {
+    .quiz__answer-text {
+        color: #ffffff;
+        background-color: colour(quiz-incorrect);
     }
 }
 
-.quiz__answer__input {
-    float: none;
-}
-
 .quiz__answer__reveal-text {
     display: block;
-    font-size: 12px;
-    padding: .5em 0em;
-}
+    color: colour(neutral-1);
+    margin-top: $gs-baseline;
+    @include fs-bodyCopy(1);
 
-.quiz__answer__icon {
-    position: absolute;
-    left: 10px;
-    top: 5px;
-    font-size: 26px;
-    color: colour(neutral-8);
+    .quiz__answer--selected & {
+        color: #ffffff;
+    }
+}
 
-    svg {
-        width: $gs-baseline;
-        height: $gs-baseline;
-        vertical-align: middle;
+.quiz--personality,
+.quiz__answer--correct,
+.quiz__answer--incorrect {
+    .quiz__answer-text {
+        padding-left: 3em;
     }
 }
 
 .quiz__end-message {
-    border-top: 1px solid colour(news-support-1);
+    border-top: 1px solid colour(neutral-4);
     background-color: colour(neutral-8);
-    padding: $gs-baseline $gs-gutter/2;
+    padding: $gs-baseline $gs-gutter;
 }
 
-.quiz__score-message {
-    @include fs-bodyCopy(3);
+.quiz__score-message,
+.quiz__bucket-message {
+    color: colour(neutral-2);
 }
 
 .quiz__score {
-    @include fs-headline(9, true);
-    color: colour(news-main-1);
-    margin-left: $gs-gutter/5;
-    vertical-align: middle;
-}
-
-.quiz__bucket-message {
-    margin-top: $gs-baseline/2;
-    margin-bottom: $gs-baseline/2;
+    @include fs-headline(9);
+    font-size: 72px;
+    line-height: 78px;
+    color: colour(guardian-brand);
+    display: block;
 }
 
 .quiz__share {
-    width: 100%;
-    // Clearfix floated children
-    overflow: hidden;
+    border-top: 1px solid colour(neutral-4);
+    padding-top: $gs-baseline;
+    margin-top: $gs-baseline;
 }
 
 .quiz__share__cta {
```

Comment:
Can this be just `.quiz__cta`? If we will need to create some other CTA we can just add `.quiz__cta--large/bright/amazing`.